src/constraints.jl

```diff
@@ -109,31 +105,28 @@ end
 function _set_core_constraint_object(
     cref::InfOptConstraintRef,
     constr::JuMP.AbstractConstraint
-    )::Nothing
+    )
     _adaptive_data_update(cref, constr, _data_object(cref))
     set_optimizer_model_ready(JuMP.owner_model(cref), false)
     return
 end
 
 # Extend _object_numbers
-function _object_numbers(cref::InfOptConstraintRef)::Vector{Int}
-    return _data_object(cref).object_nums
+function _object_numbers(cref::InfOptConstraintRef)
 end
```

Comment:
Probably didn't mean to delete `_data_object(cref).object_nums`

Reply:
Haha. Jinx.

Reply:
Yep, that's the culprit.